avaframe/ana5Utils/distanceTimeAnalysis.py

```diff
@@ -482,6 +483,17 @@ def extractFrontAndMeanValuesTT(cfgRangeTime, flowF, demHeader, mtiInfo):
         dtAnaPlots.plotRangeRaster(slRaster, rasterTransfo, cfgRangeTime['GENERAL'],
                                    mtiInfo['rangeRaster'], cLower)
 
+    if isinstance(contours, np.ndarray):
+        # transorm result field into avalanche path following coordinate system
+        # add header to result field - required for coordinate transformation
+        fieldFullComp = {'header': demHeader, 'rasterData': contours}
+        compData = aT.transform(fieldFullComp, 'field', rasterTransfo, cfgRangeTime['GENERAL']['interpMethod'])
+        contourDict = {}
+        inputs = {'simName': cfgRangeTime['GENERAL']['simHash'], 'compData': compData}
+        contourDict = oA.fetchContourLines(rasterTransfo, inputs, cfgRangeTime['PLOTS'].getfloat('ContoursThresholdValue'), contourDict)
+        log.info('contour dict created for %s' % cfgRangeTime['GENERAL']['simHash'])
+        mtiInfo['contoursSL'] = contourDict
+
```

Comment:
Could this live in a separate function? This one is already quite long...

Reply:
removed sl option